Source/Rows/Common/FieldRow.swift

```diff
@@ -268,28 +287,19 @@ open class _FieldCell<T> : Cell<T>, UITextFieldDelegate, TextFieldCell where T:
             if let titleLabel = titleLabel, let text = titleLabel.text, !text.isEmpty {
                 views["label"] = titleLabel
                 dynamicConstraints += NSLayoutConstraint.constraints(withVisualFormat: "H:[imageView]-(15)-[label]-[textField]-|", options: NSLayoutFormatOptions(), metrics: nil, views: views)
-                dynamicConstraints.append(NSLayoutConstraint(item: textField,
-                                                             attribute: .width,
-                                                             relatedBy: (row as? FieldRowConformance)?.textFieldPercentage != nil ? .equal : .greaterThanOrEqual,
-                                                             toItem: contentView,
-                                                             attribute: .width,
-                                                             multiplier: (row as? FieldRowConformance)?.textFieldPercentage ?? 0.3,
-                                                             constant: 0.0))
-            } else {
+                dynamicConstraints.append(NSLayoutConstraint(item: titleLabel, attribute: .width, relatedBy: calculatedTitlePercentage != nil ? .equal : .greaterThanOrEqual, toItem: contentView, attribute: .width, multiplier: calculatedTitlePercentage ?? 0.3, constant: 0.0))
```

Comment:
shouldn't be 'calculatedTitlePercentage ?? 0.7' instead of `calculatedTitlePercentage ?? 0.3`?